src/NationalInstruments.Analyzers/Correctness/AwaitInReadLockOrTransactionAnalyzer.cs

```diff
@@ -67,7 +67,7 @@ private void AnalyzeUsingStatement(SyntaxNodeAnalysisContext context)
             var usingStatementSyntax = (UsingStatementSyntax)context.Node;
             var containingMethodSyntax = usingStatementSyntax.Ancestors().OfType<BaseMethodDeclarationSyntax>().FirstOrDefault();
             if (containingMethodSyntax != null &&
```

Comment:
Note: sucks we can't enforce our [own conventions](https://github.com/ni/csharp-styleguide#f35-%EF%B8%8F-do-apply-consistent-formatting-on-split-statements) w/o RCS1029

Reply:
I pushed a change to fix the formatting of this